UnitTests/UnitTests.vcxproj.filters

```diff
@@ -1,36 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <Filter Include="Source Files">
```

Comment:
Why have you removed source file filters from project? Is this unitentionally?

Reply:
Oy, i am sorry, yes, is unitentionally